openapi/cluster.go

```diff
@@ -0,0 +1,48 @@
+package openapi
+
```

Comment:
Is this file auto-generated? If so, would like to know how to generate it :)

Reply:
No sadly :(, the comments are go-swagger annotations (https://goswagger.io/use/spec.html) so I'm currently trying to get used to it too.